tasks/configure-cluster.yml

```diff
@@ -0,0 +1,58 @@
+---
+
+- assert:
+    that:
+      - rabbitmq_cluster_name is defined
+      - rabbitmq_cluster_name | length > 0
+
+- name: Get cluster name
+  shell: "rabbitmqctl eval 'io:fwrite(\"~s~n\", [rabbit_nodes:cluster_name()]).' | grep -v '^ok$'"
```

Comment:
`sed '$d'` should be used to remove 'ok' from the erlang shell because the cluster name may be 'ok'.